src/components/NcRichText/NcReferencePicker/customPickerElements.js

```diff
@@ -61,4 +70,5 @@ export {
 	renderCustomPickerElement,
 	destroyCustomPickerElement,
 	isCustomPickerElementRegistered,
+	getCustomPickerElementSize,
```

Comment:
Do we need this exported actually?

Reply:
Yes, to be able to use it in
https://github.com/nextcloud/nextcloud-vue/pull/3866/files#diff-73dea770ee4a8a41a18ccabe45c3e8e129477d246dc93871d950e9240c6f5aafL42

It is not exported in index.js.